rebasehelper/plugins.py

```diff
@@ -20,16 +20,33 @@
 # Authors: Petr Hracek <phracek@redhat.com>
 #          Tomas Hozza <thozza@redhat.com>
 
+import six
+
 import pkg_resources
 
+from rebasehelper.logger import logger
+from rebasehelper.results_store import results_store
+
 
 class Plugin(object):
     name = None
+    runner = None
 
 
 class PluginLoader(object):
     @classmethod
-    def load(cls, entrypoint):
+    def load(cls, entrypoint, runner):
+        """Loads an entrypoint from setup.py.
```

Comment:
This is not exactly true, entrypoints could be defined in `setup.cfg` for example (and perhaps that would be a smart thing to do).